Modules/_sqlite/module.c

```diff
@@ -77,11 +78,23 @@ pysqlite_connect_impl(PyObject *module, PyObject *database, double timeout,
     else {
         Py_INCREF(isolation_level);
     }
-    PyObject *res = PyObject_CallFunction(factory, "OdiOiOii", database,
+    if (autocommit == NULL) {
+        autocommit = PyLong_FromLong(-1);
+        if (autocommit == NULL) {
+            Py_DECREF(isolation_level);
+            return NULL;
+        }
+    }
+    else {
+        Py_INCREF(autocommit);
+    }
+    PyObject *res = PyObject_CallFunction(factory, "OdiOiOiiO", database,
```

Comment:
This is rather silly, but there is no way around it, as long as module level functions and connection level functions are split in two different source files.

Reply:
Fixed by gh-95146